src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -280,6 +280,9 @@ public bool HasNucleus
         }
     }
 
+    [JsonIgnore]
+    public bool HasIslands => editedMicrobeOrganelles.GetIslandHexes().Any();
```

Comment:
This should use .Count > 0

Reply:
No, it shouldn't.
Any stops if it finds one island hex.
Count does not and is therefore less performant

Reply:
Does it actually do that? I suppose I haven't looked at the LINQ chain used, but I thought it would anyway compute the entire result before it can return anything? So in this case Any won't stop (much) earlier?

Reply:
https://dzone.com/articles/linq-performance-count-and-any

Reply:
In this case, I'd prefer like consistency with other non-empty checks in the codebase.
And in my opinion that doesn't apply, because *first* the really expensive recursive and set operations are ran, and *then* the result is inspected, so in the total time to run counting the results is pretty insignificant related to the expensive recursive function.

Reply:
So unless you can inline *everything* into a single LINQ line, using Any doesn't stop any of the really expensive stuff early.